src/config/constants/farms.ts

```diff
@@ -38,6 +38,16 @@ const farms: FarmConfig[] = [
   /**
    * V3 by order of release (some may be out of PID order due to multiplier boost)
    */
+  {
+    pid: 406,
```

Comment:
Next pid should be 405, is it ok to be 406?
```suggestion
    pid: 405,
```